egs/yaounde_fr/s5/RESULTS

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+
+for x in exp/*/decode*/; do [ -d "$x" ] && [[ $x =~ "$1" ]] && grep WER $x/wer_* | utils/best_wer.sh; done
+
+for x in exp/chain/*/decode*; do [ -d $x ] && [[ $x =~ "$1" ]] && grep WER $x/wer_* | utils/best_wer.sh; done
+exit 0
+
+| model | dev tgsmall | test tgsmall | devtest tgsmall | dev tgmed | test tgmed | devtest tgmed | dev tglarge | test tglarge | devtest tglarge |
+| mono | 54.58 | 25.67 | 7.86 | 76.18 | 69.64 | 52.09| | | |
+| tri1 | 32.06 | 19.14 | 3.79 | 31.98 | 27.16 | 10.06 | 29.60 | 22.28 | 5.99 |
+| tri2b | 33.02 | 19.01 | 3.85 | 33.26 | 26.36 | 9.91 | 31.48 | 21.27 | 5.53 |
+| tri3b | 26.91 | 18.85 | 3.49 | 25.90 | 24.51 | 8.51 | 23.83 | 20.01 | 4.37 |
+| chain tdnn-f | 24.02 | 17.20 | 1.96 | 22.30 | 33.66 | 16.17 | 20.14 | 18.69 | 3.33 | 
+| chain tdnn-f online | 24.21 | 17.23 | 1.96 | 22.26 | 33.72 | 16.14 | 19.10 | 32.07 | 14.74 |
```

Comment:
Did something go wrong with the last 2 results on this line?  The chain tdnnf-online with tglarge rescoring?

You don't seem to be getting as much improvement from the GMM to TDNN phase as I would normally expect.
Since there is more data, you could try increase the bottleneck dimension from 96 to 128 and reducing the l2-regularize values from 0.03 and 0.015 to 0.02 and 0.01, and reducing num-epochs from 20 to 15 (or maybe even 10, but test it).